src/utils/serializeFilterParams.js

```diff
@@ -8,33 +8,37 @@ export default function serializeFilterParams(filters, descriptions) {
   const params = {}
 
   Object.keys(filters).forEach(field => {
-    const value = filters[field]
+    const value = filters[field]?.value
     const description = descriptions[field]
-    const key = description.key
+    let key = description.key
 
     if (value === undefined)
       return
 
     switch (description.type) {
 
       case FILTER_TYPE.RANGE: {
-        if (value[0] !== null)
-          params[key + '__gte'] = value[0]
-
-        if (value[1] !== null)
-          params[key + '__lte'] = value[1]
+        if (value){
+          params[key + '__gte'] = value.min
+          params[key + '__lte'] = value.max
+        }
 
```

Comment:
This change means users now must specifiy both min & max in a search. Isn't it possible that users can want to define a lower limit but no upper limit or vice-versa?

Reply:
The other one is always set to undefined if there is one value